src/notification/notification.ts

```diff
@@ -53,6 +54,11 @@ export function sendNotification(link: Link, store: Store) {
 		sendTelegramMessage(link, store);
 	}
 
+	if (notifications.twilio.accountSid && notifications.twilio.authToken) {
+		Logger.debug('↗ sending twilio message');
+		sendTwilioMessage(link, store);
+	}
```

Comment:
Adding this causes a code complexity linting warning. Not sure on how you'd prefer this be solved. Does this function need a refactor?

Reply:
Nah, that's ok. We need to separate these out at some point. It's probably just complaining because of the line length within this function.